escrow/contracts/Escrow.cdc

```diff
@@ -227,13 +236,13 @@ pub contract Escrow {
     }
 
     init() {
-        self.CollectionStoragePath = /storage/EscrowAdmin
-        self.CollectionPrivatePath = /private/EscrowAdmin
+        self.CollectionStoragePath = /storage/EscrowLeaderboardCollection
```

Comment:
I was hoping for a reference to AllDay in the path name here

Reply:
I see! Sure I can make that change.

Reply:
Talking offline...